charts/quantum-serverless/values.yaml

```diff
@@ -3,6 +3,8 @@
 # ===================
 platform: default
 
+storageClassName: manual
+claimName: gateway-claim
```

Comment:
Do we need to also drop the `cos.claimName` value at the bottom of the values chart? Or maybe reorganize that section into a more generic PVC section?

Reply:
I didn't want to affect any current configurations.  It doesn't seem well organized now so I added new variables.  If everyone is OK, I'll reorganize the pvc configurations.

Reply:
For me it's fine too. If you think it was not well organized feel free to propose a change, Aki (meanwhile we don't loose functionality).

Reply:
Would it be possible to enable/disable COS via the storageClassName field? not necessarily for this PR, but wondering if that makes the cosEnable field redundant...

Reply:
Right now, it enables COS only for platform=ibm case.  So cosEnable is not necessary.

Reply:
For me with the changes applied by @akihikokuroda I think it's nice and resolves the conversation 👍

Reply:
@psschwei I took out `cosEnable` in the last commit.